### ISSUE TITLE
Save programs to local storage

### ISSUE THREAD

**Issue by bjpirt** [bjpirt/mirobot-ui#14]
It would be nice to be able to save a set of instructions for later use in local storage.

**Comment by silps** [comment]
I think this issue is done? https://github.com/bjpirt/mirobot-ui/blob/master/src/builder.js#L122-L128

**Comment by bjpirt** [comment]
No, this just persists it between page loads. This issue is more about being able to save multiple "files" and reopen them later. I'm currently looking at doing this on apps.mirobot.io with a generic save mechanism that any app can use. I'll then be porting this into the embedded mirobot-ui.

**Comment by silps** [comment]
I see ^_^
